src/components/Execution/Webhook.jsx

```diff
@@ -5,16 +5,21 @@ import Button from 'cozy-ui/react/Button'
 
 import { useClient } from 'cozy-client'
 
-export const Webhook = ({ hook }) => {
+export const Webhook = ({ hook, onUpdate }) => {
   const client = useClient()
 
   const [isWorking, setIsWorking] = useState(false)
   const [input, setInput] = useState('')
   const [pretrained, setPretrained] = useState(true)
 
-  const name = 
-    hook && hook.attributes ? hook.attributes.arguments.split(".")[1] : ""
-  console.log(hook, name)
+  let name = '?'
```

Comment:
Why `?` ? `I guess `let name` would be enough?

Reply:
Because at line 71, we call `name.toUpperCase()`, which fails on undefined. It could be `let name=''` but it would be less explicit that the name could not be inferred

Reply:
> it would be less explicit that the name could not be inferred
How about this:
```
let name = ''
...
<b>{name.toUpperCase() || '?'}</b>
```
This way, it is made clear that `?` is the default value to display (I initially missed that)